pro/src/components/VenueForm/Contact/__specs__/Contact.spec.tsx

```diff
@@ -0,0 +1,65 @@
+import '@testing-library/jest-dom'
+
+import { render, screen } from '@testing-library/react'
+import { Form, Formik } from 'formik'
+import React from 'react'
+import * as yup from 'yup'
+
+import { SubmitButton } from '../../../../ui-kit'
+import { DEFAULT_ACTIVITY_FORM_VALUES } from '../../Activity'
+import informationsValidationSchema from '../../Activity/validationSchema'
+import { Contact } from '../../Contact'
+import { IContactProps } from '../../Contact/Contact'
+import { IVenueFormValues } from '../../types'
+
+const renderContact = ({
+  initialValues,
+  onSubmit = jest.fn(),
+  props,
+}: {
+  initialValues: Partial<IVenueFormValues>
+  onSubmit: () => void
+  props: IContactProps
+}) => {
+  const validationSchema = yup.object().shape(informationsValidationSchema)
+
+  render(
+    <Formik
+      initialValues={initialValues}
+      onSubmit={onSubmit}
+      validationSchema={validationSchema}
+    >
+      <Form>
+        <Contact {...props} />
+        <SubmitButton isLoading={false}>Submit</SubmitButton>
+      </Form>
+    </Formik>
+  )
+}
+describe('components | Activity', () => {
```

Comment:
détail :
```suggestion
describe('Activity', () => {
```

au tout début du pass on écrivait le chemin en haut des tests, puis plus que components / Route ... 
je pense que le nom du composant suffit :)